### DISCUSSION TITLE
Practica Pasteleria

### DISCUSSION THREAD

**Discussion by LuceroLuciano** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#320] [Show and tell]
Hola explorers, veo que muchos están compartiendo sus paginas de la pastelería y yo también quiero compartirla con ustedes, no es la mejor pero seguimos trabajando en ello para mejorar.  Talvez ya la hayan visto, creo que la había compartido antes pero no estoy segura, por eso la comparto de nuevo.  : ) 
[https://pateleriahalley.azurewebsites.net/index.html ](https://pateleriahalley.azurewebsites.net/index.html )

**Comment by doguedogue** [comment]
como cuelgas tu página en ese sitio? es gratis o pagas?
